library/network_connections.py

```diff
@@ -256,7 +256,7 @@ def ValueEscape(cls, value):
         if r.match(value):
             return value
 
-        if any([ord(c) < ord(" ") for c in value]):
```

Comment:
are these changes tested on Python 2.6?

Reply:
> are these changes tested on Python 2.6?

I tested them on a centos-6 image using python 2.6

Reply:
This is the error from ansible-test:
```
Running sanity test "pylint"
ERROR: Found 8 pylint issue(s) which need to be resolved:
ERROR: plugins/module_utils/network_lsr/argument_validator.py:667:29: use-a-generator: Use a generator instead 'any(a for a in result['address'] if a['family'] == socket.AF_INET6)'
ERROR: plugins/module_utils/network_lsr/argument_validator.py:698:79: use-a-generator: Use a generator instead 'any(a for a in result['address'] if a['family'] == socket.AF_INET)'
ERROR: plugins/module_utils/network_lsr/utils.py:223:11: use-a-generator: Use a generator instead 'any(c == '/' or c == ':' or c.isspace() for c in ifname)'
ERROR: plugins/modules/network_connections.py:259:11: use-a-generator: Use a generator instead 'any(ord(c) < ord(' ') for c in value)'
ERROR: plugins/modules/network_connections.py:473:21: consider-using-generator: Consider using a generator instead 'list(a for a in ip['address'] if a['family'] == socket.AF_INET)'
ERROR: plugins/modules/network_connections.py:474:21: consider-using-generator: Consider using a generator instead 'list(a for a in ip['address'] if a['family'] == socket.AF_INET6)'
ERROR: plugins/modules/network_connections.py:1029:21: consider-using-generator: Consider using a generator instead 'list(a for a in ip['address'] if a['family'] == socket.AF_INET)'
ERROR: plugins/modules/network_connections.py:1030:21: consider-using-generator: Consider using a generator instead 'list(a for a in ip['address'] if a['family'] == socket.AF_INET6)'
```

Reply:
Although for the `consider-using-generator` errors I think using a list comprehension is more pythonic than using `list(a generator expression)`